.github/workflows/ci.yml

```
@@ -0,0 +1,183 @@
+        with:
+          go-version: '1.24'
+          cache: true
+
+      - name: Install dependencies
+        run: go mod download
+
+      - name: Install staticcheck
+        run: go install honnef.co/go/tools/cmd/staticcheck@latest
+
+      - name: Install golangci-lint
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:101: Quote this to prevent word splitting

(shellcheck)